microcosm_flask/tests/swagger/test_definitions.py

```diff
@@ -166,6 +166,7 @@ def test_build_swagger():
                     "email": {
                         "type": "string",
                         "format": "email",
+                        "x-nullable": True,
```

Comment:
Does this mean we have to regenerate all of our swagger schemas so this gets generated into them or is this already present in our schemas?

Reply:
We will need to regenerate.